README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
```

Comment:
when i am swapping the line badge is not visible 
![screenshot from 2017-07-23 18-54-36](https://user-images.githubusercontent.com/10944610/28499659-7ce48c5e-6fd8-11e7-9eab-c9329dce39d9.png)

![screenshot from 2017-07-23 18-54-44](https://user-images.githubusercontent.com/10944610/28499660-7dda8e42-6fd8-11e7-87a5-62a579c800b8.png)

What should i do @vaibhavsingh97